pages/common/vue-init.md

```diff
@@ -0,0 +1,16 @@
+# vue init
+
+> Legacy project initialization subcommand of the Vue.js framework.
+> More information: <https://cli.vuejs.org/guide/creating-a-project.html#pulling-2-x-templates-legacy>.
+
+- Create a new Vue.js project:
```

Comment:
I'd mention the template parameter in the description, to make it explicit, and to match the structure of the following two examples. For example:

```suggestion
- Create a new Vue.js project using one of the default templates:
```

Reply:
Also, the language should be similar in all three cases, so instead of:
>\- Create a new Vue.js project...
>\- Create a new project...
>\- Create a project...

all three examples should start the same way, e.g.
>\- Create a new project...